### PR TITLE
For #6167 Refactor FingerPrint to Biometric

### PR DESCRIPTION
**Pull request by iorgamgabriel**
https://github.com/mozilla-mobile/focus-android/issues/6167

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -242,6 +242,7 @@ dependencies {
     implementation "org.mozilla.components:browser-state:${AndroidComponents.VERSION}"
     implementation "org.mozilla.components:browser-toolbar:${AndroidComponents.VERSION}"
 
+
```
Comment (ionutbedregeanu) [inline app/build.gradle:245]: nit: Do we need this extra space?

Comment (iorgamgabriel) [reply app/build.gradle]: Done

**Comment by ionutbedregeanu** [comment]
I think we can land it as it is and create a new ticket for https://github.com/mozilla-mobile/focus-android/pull/7198#issuecomment-1166974668

**Review by ionutbedregeanu — APPROVED** [review]

**Comment by iorgamgabriel** [comment]
Thank you ! @ionutbedregeanu

**Pull request merged**